File: 03-RomanNumerals/csharp-dotnetframework/RomanNumeralsTest/RomanToIntegerConverterTest.cs
using NUnit.Framework;
using RomanNumerals;

namespace Katas
{
    [TestFixture]
    public class RomanToIntegerConverterTest
    {
        [Test]
        [Ignore("Remove this line when you're ready to run the tests")]
        public void ShouldConvertsXVIto16()
        {
            var converter = new RomanToIntegerConverter();

            var actualResult = converter.Convert("XVI");
            var expectedResult = 16;

            Assert.AreEqual(expectedResult, actualResult);
        }
    }
}

File: 03-RomanNumerals/java/src/test/java/RomanToIntegerConverterTest.java
import static org.junit.Assert.assertEquals;

import org.junit.Ignore;
import org.junit.Test;

public class RomanToIntegerConverterTest {

  @Test
  @Ignore("Remove this line when you are ready to run this test")
  public void canConvertXVItoSixteen() throws Exception {
    RomanToIntegerConverter converter = new RomanToIntegerConverter();

    assertEquals(1, converter.convertToInteger("XVI"));
  }

}
